<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="setInfo"/>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QDateEdit" name="setDateBirth"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="birthDay">
      <property name="text">
       <string>Дата рождения:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="status">
      <property name="text">
       <string>Семейный статус:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="setPatronomyc"/>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="setName"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="info">
      <property name="text">
       <string>Дополнительная информация:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="sex">
      <property name="text">
       <string>Пол:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="name">
      <property name="text">
       <string>Имя:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="clear">
      <property name="text">
       <string>Очистить форму</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="setSurname"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="patronomyc">
      <property name="text">
       <string>Отчество:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="surname">
      <property name="text">
       <string>Фамилия:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="setStatus"/>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="save">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="setSex"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
